flogo commands ends in error

After taking the latest cli [@8cf8b43](https://github.com/project-flogo/cli/tree/8cf8b43d97113753521feff53b0db81756867b31) using `go get -u github.com/project-flogo/cli/...`, I get the following error for any flogo command I use:

```bash
C:\Users\apandura\Desktop\AnandDocuments\workspace\web-ui>flogo
panic: "cv" shorthand is more than one ASCII character

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc0000da300, 0xc000053220)
        C:/Users/apandura/Desktop/AnandDocuments/goworkspace-test/src/github.com/spf13/pflag/flag.go:852 +0x77d
github.com/spf13/pflag.(*FlagSet).VarPF(0xc0000da300, 0x8e4700, 0xba5ba0, 0x86e6cd, 0xb, 0x86c206, 0x2, 0x876873, 0x1c, 0xc000053180)
        C:/Users/apandura/Desktop/AnandDocuments/goworkspace-test/src/github.com/spf13/pflag/flag.go:819 +0x112
github.com/spf13/pflag.(*FlagSet).VarP(0xc0000da300, 0x8e4700, 0xba5ba0, 0x86e6cd, 0xb, 0x86c206, 0x2, 0x876873, 0x1c)
        C:/Users/apandura/Desktop/AnandDocuments/goworkspace-test/src/github.com/spf13/pflag/flag.go:825 +0x95
github.com/spf13/pflag.(*FlagSet).StringVarP(0xc0000da300, 0xba5ba0, 0x86e6cd, 0xb, 0x86c206, 0x2, 0x0, 0x0, 0x876873, 0x1c)
        C:/Users/apandura/Desktop/AnandDocuments/goworkspace-test/src/github.com/spf13/pflag/string.go:42 +0xb1
github.com/project-flogo/cli/commands.init.1()
        C:/Users/apandura/Desktop/AnandDocuments/goworkspace-test/src/github.com/project-flogo/cli/commands/create.go:45 +0x10a
```

--- a/commands/create.go
+++ b/commands/create.go
@@ -42,6 +42,6 @@ var CreateCmd = &cobra.Command{
 
 func init() {
 	CreateCmd.Flags().StringVarP(&flogoJsonPath, "file", "f", "", "path to flogo.json file")
-	CreateCmd.Flags().StringVarP(&coreVersion, "coreVersion", "cv", "", "specify core library version")
+	CreateCmd.Flags().StringVarP(&coreVersion, "coreVersion", "c", "", "specify core library version")
 	rootCmd.AddCommand(CreateCmd)
 }

--- a/commands/create.go
+++ b/commands/create.go
@@ -42,6 +42,6 @@ var CreateCmd = &cobra.Command{
 
 func init() {
 	CreateCmd.Flags().StringVarP(&flogoJsonPath, "file", "f", "", "path to flogo.json file")
-	CreateCmd.Flags().StringVarP(&coreVersion, "coreVersion", "c", "", "specify core library version")
+	CreateCmd.Flags().StringVarP(&coreVersion, "cv", "", "", "specify core library version")
 	rootCmd.AddCommand(CreateCmd)
 }

--- a/commands/install.go
+++ b/commands/install.go
@@ -44,7 +44,7 @@ var installCmd = &cobra.Command{
 
 func init() {
 	installCmd.Flags().StringVarP(&localContrib, "localContrib", "l", "", "Specify local Contrib")
-	installCmd.Flags().StringVarP(&contribBundle, "contribBundle", "cb", "", "Specify ContribBundle")
+	installCmd.Flags().StringVarP(&contribBundle, "contribBundle", "c", "", "Specify ContribBundle")
 	rootCmd.AddCommand(installCmd)
 
 }

--- a/commands/install.go
+++ b/commands/install.go
@@ -16,6 +16,7 @@ var installCmd = &cobra.Command{
 	Short: "install a flogo contribution",
 	Long:  "Installs a flogo contribution",
 	Run: func(cmd *cobra.Command, args []string) {
+
 		if contribBundle != "" {
 			err := api.InstallContribBundle(common.CurrentProject(), contribBundle)
 			if err != nil {
@@ -44,7 +45,7 @@ var installCmd = &cobra.Command{
 
 func init() {
 	installCmd.Flags().StringVarP(&localContrib, "localContrib", "l", "", "Specify local Contrib")
-	installCmd.Flags().StringVarP(&contribBundle, "contribBundle", "c", "", "Specify ContribBundle")
+	installCmd.Flags().StringVarP(&contribBundle, "file", "f", "", "Specify ContribBundle file")
 	rootCmd.AddCommand(installCmd)
 
 }

